### ISSUE TITLE
No documentation for DCB directive

### ISSUE THREAD

**Issue by neildavis** [prb28/vscode-amiga-assembly#308]
Firstly, thanks for this amazing extension which I'm really enjoying using to learn 68K/Amiga asm.

I found a problem in the automatic documentation (when you hover over an instruction and the documentation pops up). The `DCB` directive is not supported. e.g:

```asm
		SECTION	Data,BSS_C 

Bitplanes:
		DCB.B	(352*32*4)/8
		DC.B	0     ; only here for example
```

Hover over `DC.B` and the docs for `DC` pop up from [here](/prb28/vscode-amiga-assembly/blob/master/docs/directives/dc.md)

Hover over `DCB.B` and nothing happens.

This is because there is no `DCB.md` in [`/docs/directives`](/prb28/vscode-amiga-assembly/blob/master/docs/directives). There is a [`DBC.md`](/prb28/vscode-amiga-assembly/blob/master/docs/directives/dbc.md) which actually docs `DCB` but with the wrong title. This is also reflected in the [TOC](/prb28/vscode-amiga-assembly/blob/master/docs/toc.md)

This issue is also in [m68k-instructions-documentation](/prb28/m68k-instructions-documentation) for which I raised this [pull request](/prb28/m68k-instructions-documentation/pull/27)

I would raise a similar PR here but I'm assuming this repo is built to include [m68k-instructions-documentation](/prb28/m68k-instructions-documentation) somehow, so I'll leave that to the maintainer(s).

**Comment by prb28** [comment]
Thanks for PR, You're right about this mistake.

**Comment by prb28** [comment]
fixed in the 1.8.11 pre-release.
